lib/src/controllers/radial_controller.dart

```diff
@@ -1,16 +1,51 @@
+import 'dart:math';
+
+import 'package:chart_it/src/animations/chart_animations.dart';
+import 'package:chart_it/src/charts/data/core/radial/radial_data.dart';
+import 'package:chart_it/src/charts/data/core/radial/radial_mixins.dart';
+import 'package:chart_it/src/charts/data/pie/pie_series.dart';
+import 'package:chart_it/src/charts/painters/radial/pie_painter.dart';
+import 'package:chart_it/src/charts/painters/radial/radial_painter.dart';
+import 'package:chart_it/src/extensions/data_conversions.dart';
+import 'package:chart_it/src/extensions/primitives.dart';
 import 'package:flutter/material.dart';
 
-class RadialController extends ChangeNotifier {
-  final double minValue;
-  final double maxValue;
+class RadialController extends ChangeNotifier
+    with RadialDataMixin, ChartAnimationsMixin<RadialSeries> {
+  final Map<RadialSeries, RadialConfig> _seriesConfigs = {};
+  final Map<Type, RadialPainter> painters = {};
+
+  @override
+  double minValue = 0.0;
+  @override
+  double maxValue = 0.0;
```

Comment:
Should we reset this values as well just like the other controller?

Reply:
done